travel-planner/travel-planner-backend/routes/maps.js

```diff
@@ -5,9 +5,62 @@ const bodyParser = require("body-parser");
 router.use(bodyParser.json());
 const Parse = require("parse/node");
 const axios = require("axios");
+// const {Client} = require("@googlemaps/google-maps-services-js");
 
 // create delete pin endpoint?
```

Comment:
while you here, can you also remove this question? we shouldn't be adding comments that do not help understand the code. The caveat to that is adding a comment like "TODO (issue: N) : Add functionality X" which would then be tracked by a codehub issue and help people know this will change in this direction